examples/pytorch/hetgnn/evaluation.py

```diff
@@ -30,15 +30,15 @@ def evaluate_node_classification(args, train_num, test_num, datafile_dir):
     train_features = None
     train_target = None
 
-    test_data_f = str(datafile_dir.join("test_class_feature.txt"))
-    test_data = load_data(test_data_f, args.embed_d + 2, test_num)
+    test_data_f = str(config["data_dir"] + ("/test_class_feature.txt"))
```

Comment:
The / in test_class_feature is not cross platform, should be os.path.join(a, b). Applies other spots as well.